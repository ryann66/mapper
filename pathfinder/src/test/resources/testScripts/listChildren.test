CreateGraph g1
AddNode g1 alpha
AddNode g1 bravo
AddNode g1 charlie
ListChildren g1 alpha
AddEdge g1 alpha bravo 3
ListChildren g1 alpha
#add to back
AddEdge g1 alpha charlie 1
ListChildren g1 alpha
#add to front
AddEdge g1 alpha alpha 2
ListChildren g1 alpha
#test secondary sort
AddEdge g1 alpha bravo 4
ListChildren g1 alpha