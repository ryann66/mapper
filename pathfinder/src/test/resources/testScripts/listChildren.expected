created multigraph g1
added node alpha to g1
added node bravo to g1
added node charlie to g1
the children of alpha in g1 are:
added edge 3.000 from alpha to bravo in g1
the children of alpha in g1 are: bravo(3.000)
#add to back
added edge 1.000 from alpha to charlie in g1
the children of alpha in g1 are: bravo(3.000) charlie(1.000)
#add to front
added edge 2.000 from alpha to alpha in g1
the children of alpha in g1 are: alpha(2.000) bravo(3.000) charlie(1.000)
#test secondary sort
added edge 4.000 from alpha to bravo in g1
the children of alpha in g1 are: alpha(2.000) bravo(3.000) bravo(4.000) charlie(1.000)